client/src/pages/Login/Login.tsx

```diff
@@ -31,30 +45,22 @@ const Login = () => {
       exit={{ opacity: 0, y: -20 }}
     >
       <FormSection>
-        <LoginForm onSubmit={(e: React.FormEvent) => handleSubmit(e)}>
+        <LoginForm onSubmit={handleSubmit(submitForm)}>
           <Title>Login</Title>
           <InputGroup>
             <Label>Email</Label>
-            <Input
-              type="email"
-              value={email}
-              onChange={(e: React.FormEvent) =>
-                setEmail((e.target as HTMLInputElement).value)
-              }
-            />
+            <Input type="text" {...register("email")}/>
+            {errors.email && errors.email?.message && (
+            <p> {errors.email.message}</p>)}
```

Comment:
I think we can create another container called ErrorContainer here here to put this message in - you can see my TransactionItemAdder.styles.ts for an example and how I implemented this.